cdc/sink/codec/json.go

```diff
@@ -112,6 +112,32 @@ func formatColumnVal(c column) column {
 				log.Panic("invalid column value, please report a bug", zap.Any("col", c), zap.Error(err))
 			}
 		}
+	case mysql.TypeFloat, mysql.TypeDouble:
```

Comment:
Are they bugfix?

Reply:
Yes, I realized some types were not handled more column types were added in unit tests.